CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.1.0 - 2019-10-14
+
+- SMTPAdapter now does not append `Bcc` and `Cc` headers to the body if there is not any provided.
```

Comment:
We should add the `([#<pr-id>])` at the end of the line and add the footnote at the end of the CHANGELOG section.

You can find an example in version `2.0.0` below